Bastien Olivier Dijkstra/Clean.cs

```diff
@@ -0,0 +1,58 @@
+﻿using UnityEngine;
+
+public class Clean : MonoBehaviour
+{
+
+    public float range = 100f;
+
+    private RaycastHit selected;
+    private RaycastHit lastSelected;
+
+    private AudioSource cleanSoundSource;
+
```

Comment:
Missing the dubble white spaces (enters).

Reply:
Thank you for reminding me of the rules, but this was the unedited version of my homework. It wouldn't push only a folder. But I just reformated my file, your welcome to take a look.